### ISSUE TITLE
Android/Spring codebase unification

### ISSUE THREAD

**Issue by typelogic** [idpass/idpass-lite-java#10]
This is a long-term improvement initiatives to have a common code base. The Android cobedase is a slightly modified version due to the use of `BufferedImage` in this codebase.

**Comment by typelogic** [comment]
Findings is that, `Card::open(BufferedImage bufferedImage)` may not be part of the `Card` method and can be removed. The outer system (either Android or Desktop) already has the `byte[]` array after the outer system invoke its `QR scanning`. 

As for `BufferedImage Card::asQRCode()` method is only  used in unit tests. In `mosip-idpass-print`,  the  method `String Card::asQRCodeSVG()` is the one that is really used. 

In summary, my current findings is that:
- `idpass-lite-java` consumes image input as `byte[]`  
- `idpass-lite-java` produces an image output in `SVG` format.

**Comment by typelogic** [comment]
Update notes:
- Removing `BufferedImage` was a straight-forward. What was not straight-forward was the encounter of Android `protobuf-lite` not supporting reflection. Therefore, I added `merge_CardDetails` inside `libidpasslite.so` 
- How to have a common 1 jar that works both for Android and Spring Boot? As I recall, Android project explicitly and manually put `libidpasslite.so` under `jniLibs` folder and this means I should not bundle it in the jar. Furthermore, the `idpasslite.dll` and `libidpasslite.so` comes with the jar which Android does not need.

**Comment by typelogic** [comment]
Fixed at https://github.com/idpass/idpass-lite-java/commit/13a74ef688999df78047436ca5caf5b7f2545fcb
